cocotb/_gpi_embed.py

```diff
@@ -0,0 +1,41 @@
+import logging
+import os
+import importlib
+from functools import reduce
+
+
+def _filter_from_c(logger_name, level):
+    return logging.getLogger(logger_name).isEnabledFor(level)
+
+
+def _log_from_c(logger_name, level, filename, lineno, msg, function_name):
+    """
+    This is for use from the C world, and allows us to insert C stack
+    information.
+    """
```

Comment:
```suggestion
    """Log from the C world, allowing to insert C stack information."""
```
(I know this is from original code, but now is a chance to clean this up.)